src/hooks/useOperationModal/Modal/SupplyForm/__tests__/__snapshots__/indexIntegratedSwap.spec.tsx.snap

```diff
@@ -2,59 +2,59 @@
 
 exports[`hooks/useSupplyWithdrawModal/Supply - Feature flag enabled: integratedSwap > displays correct swap details 1`] = `
 <div
-  class="css-sdj0qa-SupplyFormUi"
+  class="css-1o151nw-SupplyFormUi"
   data-testid="swap-details"
 >
   <div
-    class="css-u34kla-SwapDetails"
+    class="css-s1vrvo-SwapDetails"
   >
     <div
       class="css-wkfae5-LabeledInlineContent"
     >
       <span
-        class="MuiTypography-root MuiTypography-body1 css-1degdl8-MuiTypography-root-LabeledInlineContent"
+        class="MuiTypography-root MuiTypography-body1 css-16extv5-MuiTypography-root-LabeledInlineContent"
```

Comment:
Same comment on this diff, it isn't as useful as snapshots because you can't understand the impact of the css changes from this diff

Reply:
Same as before, we indeed should not test css differences